src/bentoml/_internal/runner/runner_handle/remote.py

```diff
@@ -297,18 +297,22 @@ async def is_ready(self, timeout: int) -> bool:
         # default kubernetes probe timeout is also 1s; see
         # https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes
         aio_timeout = aiohttp.ClientTimeout(total=timeout)
-        async with self._client.get(
-            f"{self._addr}/readyz",
-            headers={
-                "Bento-Name": component_context.bento_name,
-                "Bento-Version": component_context.bento_version,
-                "Runner-Name": self._runner.name,
-                "Yatai-Bento-Deployment-Name": component_context.yatai_bento_deployment_name,
-                "Yatai-Bento-Deployment-Namespace": component_context.yatai_bento_deployment_namespace,
-            },
-            timeout=aio_timeout,
-        ) as resp:
-            return resp.status == 200
+        try:
+            async with self._client.get(
+                f"{self._addr}/readyz",
+                headers={
+                    "Bento-Name": component_context.bento_name,
+                    "Bento-Version": component_context.bento_version,
+                    "Runner-Name": self._runner.name,
+                    "Yatai-Bento-Deployment-Name": component_context.yatai_bento_deployment_name,
+                    "Yatai-Bento-Deployment-Namespace": component_context.yatai_bento_deployment_namespace,
+                },
+                timeout=aio_timeout,
+            ) as resp:
+                return resp.status == 200
+        except asyncio.TimeoutError:
+            logger.error("Timeout exception caught while getting '/readyz'", exc_info=True)
```

Comment:
Maybe "Timed out waiting for runner to be ready"?

Also, we probably shouldn't print exception info, and I'm not sure this should be an error. Maybe info?

Reply:
Yeah I'm just following the exception format in `handle_triton_exception`. Maybe we can use `warn`.